### DISCUSSION TITLE
Countup Event

### DISCUSSION THREAD

**Discussion by urbs2005** [cpvalente/ontime#489] [Q&A]
First of all, this has been an amazing software to dig into, and has been really helpful in educational theatre.

I think I just haven't found the exact settings I need for a certain use case.  I'd like to have an event that starts whenever I press start and counts down to a certain time.  It would then overcount until the next event is started.

I realize that the countdown view is ideally suited for this, but I need the other features of the timer screen, such as messages.

Any help is greatly appreciated!

Chris

**Comment by cpvalente** [comment]
Hi,

Really happy to hear this. I have been trying to see how we can make ontime better suited for theater / opera. Perhaps you would be available to help us?

As for your question:
I am very onboard with it, but would like to understand the use case.
How about a timer-type `count-to-end` that will always show a countdown to the end of the event, no matter when it was started?


If you have availability to talk about how do you use the app, what painpoints it solves and what else it could do, get in touch at mail @ ontime.no

**Comment by urbs2005** [comment]
I'm happy to help if I can.  I can tell you that I was using stagetimer.io in
its free version prior to finding your software, so some of the things I'm
looking for are things that were easy there.

The particular use case at the moment that I've been working with is a
magic show:  The timer is controlled from the booth, there is an ipad
located on stage for the talent, and another computer backstage for an fx
tech.

The timer is often setup towards the beginning of the call, and I would
like it to count up to the time we open doors.  It then is supposed to
count down the 35 minute audience period, and wait for a start command
before beginning a count up timer.

I think possibly a count-to-end timer might work, but I'm not sure I
quite understand the concept.

I'm also hoping to use the app during plays and musicals to run timers to
dressing rooms and green rooms, much in a similar way.

Please let me know if you would like clarification on anything I've
mentioned, as I'm not sure if I'm explaining it the best way!

Thanks so much for all of your work on this - I've really enjoyed getting
to know the software.

Chris

**Comment by cpvalente** [comment]
I am slightly confused now, we already have count-up and count-down events as you describe.

Is it that you want to count not to the next event but to another event in an arbitrary order in the rundown?

Would the correct feature request then be the addition of messages to countdown screen?

It could be helpful if you could email a sample project file and some notes on the issue and I can try and help from there

**Reply by urbs2005** [reply]
I'll be back in tomorrow and get a project file.  It's possible what I'm looking for already exists and I'm just not using the right combination of settings.  The countdown screen does part of what I'm looking for, but changing from the countdown to event timer screen is not something I want to have to do.  I'll try to make it a little clearer with a file!

**Comment by cpvalente** [comment]
For people coming here in future.
At this moment, Ontime's timer is attached to the schedule. ie: for a timer, you need a scheduled event

For any event, you can set the timer as count-up, this is a style override that is only applied in the presenter view
If you need a timer that is not attached to the schedule, you can leverage the auxiliary timer
https://docs.getontime.no/features/aux-timer/
